runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java

```diff
@@ -412,24 +412,36 @@ private void getItemPrice(int id, ItemDefinition itemComposition, int quantity)
 								.append(ChatColorType.HIGHLIGHT)
 								.append(QuantityFormatter.formatNumber(gePrice * finalQuantity));
 
-							if (osbresult != null)
+							if (finalQuantity > 1)
 							{
 								message
-									.append(ChatColorType.NORMAL)
-									.append(" OSB  ")
-									.append(ChatColorType.HIGHLIGHT)
-									.append(QuantityFormatter.formatNumber(osbresult.getOverall_average() * finalQuantity));
+										.append(ChatColorType.NORMAL)
+										.append(" (")
+										.append(ChatColorType.HIGHLIGHT)
+										.append(QuantityFormatter.formatNumber(gePrice))
+										.append(ChatColorType.NORMAL)
+										.append("ea)");
 							}
 
-							if (finalQuantity > 1)
+							if (osbresult != null)
 							{
+								int osbAverage = osbresult.getOverall_average();
```

Comment:
I'd check if this was 0 as well, cause if osb price is unknown having that in the message isn't helpful either.